roadmap/2023.md

```diff
@@ -0,0 +1,42 @@
+# Roadmap 2023
+
+This is the roadmap planned by the Integration team at Meilisearch for 2023.
+
+Nothing is definitive: the roadmap can be adapted depending on the need of the users and the company.
+
+## 2023: the Main Lines
+
+#### Objectives
```

Comment:
I would add "Focus on the impact: spend time updating SDKs that are mostly used, and work with the community for the other ones", or something similar